api/v1/contexts/PKPContextHandler.php

```diff
@@ -436,7 +445,7 @@ public function editTheme($slimRequest, $response, $args)
         }
 
         $userRoles = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_USER_ROLES);
-        if (!$requestContext && !in_array(Role::ROLE_ID_SITE_ADMIN, $userRoles)) {
+        if (!in_array(Role::ROLE_ID_SITE_ADMIN, $userRoles)) {
```

Comment:
@ewhanson This now prevents Journal Managers who are not Site Admins from changing the journal theme. I've filed it here: https://github.com/pkp/pkp-lib/issues/8909
Can you have a look? Should be a quick fix.